docs/troubleshooting/windows-pv-tools.md

```diff
@@ -60,4 +60,19 @@ It's possible that some antivirus blocks the end of the installation of the PV d
 
 ### Solution
 
-Simply pausing the agent and doing a reboot will make the XenTools to install succesfully. After a succesfull installation, enabling the SentinelOne agent again is possible without any other issues regarding the tools or drivers.
\ No newline at end of file
+Simply pausing the agent and doing a reboot will make the XenTools to install succesfully. After a succesful installation, enabling the SentinelOne agent again is possible without any other issues regarding the tools or drivers.
```

Comment:
```suggestion
Simply pausing the agent and doing a reboot will make the XenTools to install successfully. After a successful installation, enabling the SentinelOne agent again is possible without any other issues regarding the tools or drivers.
```

Reply:
Sorry, I just realized this was in the context of the patch, not one of your changes.

Reply:
No problem, I can add it as a small fix.